src/main/java/ch/njol/skript/effects/EffCancelItemUse.java

```diff
@@ -0,0 +1,77 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Cancel Active Item Use")
+@Description(
+	"Interrupts the action an entity is trying to complete. " +
+	"For example, interrupting eating, or drawing back a bow."
+)
+@Examples({
+	"on damage of player:",
+	"\tif victim's active tool is a bow:",
+	"\t\tinterrupt player's active item use"
+})
+@Since("INSERT VERSION")
+@RequiredPlugins("Paper 1.16.5 or newer")
```

Comment:
Is it 1.16 or 1.16.5 specifically?
```suggestion
@RequiredPlugins("Paper 1.16.5+")
```

Reply:
I tried to find the exact paper version it was added, but all I could tell was roughly around 1.16.2/1.16.3
However, Skript only supports the latest versions of the releases, so I put 1.16.5

Reply:
Skript supported versions is different from this required version for a specific version, so it's better to mention the correct version when this  was added

Reply:
Alright, I'll do a bit more digging then